app/export.go

```
@@ -174,28 +176,28 @@
 	// iterate through redelegations, reset creation height
-	app.StakingKeeper.IterateRedelegations(
-		ctx,
-		func(_ int64, red stakingtypes.Redelegation) (stop bool) {
-			for i := range red.Entries {
-				red.Entries[i].CreationHeight = 0
-			}
-			app.StakingKeeper.SetRedelegation(ctx, red)
-			return false
-		},
-	)
+	app.StakingKeeper.IterateRedelegations(ctx, func(_ int64, red stakingtypes.Redelegation) (stop bool) {
```

golangci-lint (1.62.2)

Error return value of `app.StakingKeeper.IterateRedelegations` is not checked

(errcheck)